java/arcs/core/data/Recipe.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.core.data
+
+import arcs.core.type.Type
+
+/** Representation of recipe in an Arcs manifest. */
+data class Recipe(val name: String) {
+    /** Definition of a handle in a recipe. */
+    data class Handle(
+        val name: String,
+        val fate: Fate,
+        val storageKey: String,
+        val type: Type,
+        val associatedHandles: List<String>
```

Comment:
Similar Here, at some point this should be a reference to a Handle, not a string.

Reply:
Oh yes. This is stop gap for me to push forward. I meant to add a TODO, but seems to have forgotten about it. Will add it once I go through the internal readability.